lib/Basic/Subprocess.cpp

```diff
@@ -528,19 +534,17 @@ void llbuild::basic::spawnProcess(
     }
 
     if (control.shouldRelease()) {
-      // Close the read end of the release pipe.
-      ::close(releasePipe[0]);
-
       releaseFn([
```

Comment:
Might be worth either putting a comment here, or at the other close site, explaining why we close when we do.